blog/2022-08-08-bluetooth-api-changes.md

```diff
@@ -0,0 +1,89 @@
+---
+author: John Carr
+authorURL: https://github.com/Jc2k
+title: "Bluetooth passive sensor API changes for 2022.9"
+---
+
+For Home Assistant Core 2022.9 we have changesd the `PassiveBluetoothProcessorCoordinator` and `PassiveBluetoothDataProcessor` bluetooth API's to make `PassiveBluetoothProcessorCoordinator` responsible for parsing. The coordinator then pushes parsed data to `PassiveBluetoothDataProcessor` instances.
+
+PassiveBluetoothProcessorCoordinator now takes a mandatory update_method callback that receives bluetooth advertisements (in the form of BluetoothServiceInfoBleak) and returns the data that should be handed off to any subscribed PassiveBluetoothDataProcessor:
+
+```diff
```

Comment:
I don't think we should include the diff. Just show an example of the new interface.

Reply:
Just pushed a change. I'm not the best at explaining things, so was hard to know whether this was too much or not enough of an example.

Reply:
Looks good!